### ISSUE TITLE
On release, update a tag/url on CDN always pointing to latest.

### ISSUE THREAD

**Issue by dconnolly** [videojs/video.js#2109]
So that we can always use latest on videojs.com, etc. 
See videojs/videojs.com#24

**Comment by heff** [comment]
So we used to have this with 3.0 and it was scary as hell every time we pushed a change, thinking we could break a ton of sites out there. See jQuery's issue too: http://blog.jquery.com/2014/07/03/dont-use-jquery-latest-js/

With 4.0 we switched to having minor versions auto-update with patches only, e.g. /4.5/video.js would get 4.5.1, 4.5.2, etc., but 4.4 would not get any of those updates.

We could increase this to major versions, so /5/video.js would always be the latest 5.x release. Even then I'd be tempted to add a built in console warning that says "don't use in production".

The other thing Matt talked about doing was publishing the package.json to the CDN, and checking that for the latest version.

I could go either way on those.
